core/src/main/java/ai/timefold/solver/core/impl/domain/variable/descriptor/ShadowVariableDescriptor.java

```diff
@@ -64,6 +64,11 @@ public boolean hasVariableListener() {
         return true;
     }
 
+    /**
+     * return true if the source variable is a basic variable; otherwise, return false.
+     */
+    public abstract boolean isListVariableSource();
```

Comment:
The name of the method suggests the exact opposite of what the Javadoc says.

Reply:
Yep. It is wrong.